spec/offchain.tex

```diff
@@ -110,26 +112,31 @@ \subsection{Variables}
 \begin{itemize}
 	\item $\hatv$: Last seen open state version.
 	\item $\hats$: Sequence number of latest seen snapshot.
-	\item $\hatSigma \in {(\tyNatural \times \tyBytes)}^{*}$: Accumulator of
-	      signatures of the latest seen snapshot, indexed by parties.
+	\item $\hatSigma \in {(\tyNatural \times \tyBytes)}^{*}$:
+		Accumulator of signatures of the latest seen snapshot, indexed by parties.
 	\item $\hatmL$: UTxO set representing the local ledger state resulting from
-	      applying $\hatmT$ to $\bar{S}.U$ to validate requested transactions.
+		applying $\hatmT$ to $\bar{S}.U$ to validate requested transactions.
 	\item $\hatmT \in \mT^{*}$: List of transactions applied locally and pending
-	      inclusion in a snapshot (if this party is the next leader).
-	\item $\tx_{\omega} \in \mathcal{T}^{?}$: Pending decommit transaction, whose outputs are to be decommitted from the head. May be $\bot$ if nothing to decommit.
+		inclusion in a snapshot (if this party is the next leader).
+	\item \red{$U_\alpha \in \gcUTXOset$: Pending increment UTxO to be deposited to the head. Can be bottom if nothing is pending to be deposited.}
+	\item $U_\omega \in \gcUTXOset$: Pending decrement UTxO to be withdrawn from the head. Can be bottom if nothing is pending to be withdrawn.
 	\item $\bar{\mc S}$: Snapshot object of the latest confirmed snapshot which contains:
-	      \begin{center}
-		      \begin{tabular}{|l|l|}\hline
-			      $\bar{\mc S}.v$            & snapshot version                                              \\ \hline
-			      $\bar{\mc S}.s$            & snapshot number                                               \\ \hline
-			      $\bar{\mc S}.T$            & set of transactions relating this snapshot to its predecessor \\ \hline
-			      $\bar{\mc S}.U$            & snapshotted UTxO set                                          \\ \hline
-			      $\bar{\mc S}.\tx_{\omega}$ & optional decommit transaction                                 \\ \hline
-			      $\bar{\mc S}.\sigma$       & multisignature                                                \\ \hline
-		      \end{tabular}
-	      \end{center}
-	      The function $\Sno(v,n,T,U,\tx_{\omega})$ initializes a snapshot object with no multi-signature set.
-	      \todo{add types?}
+		\begin{center}
+			\begin{tabular}{|l|l|}\hline
+				$\bar{\mc S}.v$         & snapshot version \\ \hline
+				$\bar{\mc S}.s$         & snapshot number \\ \hline
+				$\bar{\mc S}.T$         & set of transactions relating this snapshot to its predecessor \\ \hline
+				$\bar{\mc S}.U$         & snapshotted UTxO set \\ \hline
+				\red{$\bar{\mc S}.U_\alpha$}  & \red{peding UTxO to commit} \\ \hline
+				\red{$\bar{\mc S}.U_\omega$}  & \red{peding UTxO to decommit} \\ \hline
+				$\bar{\mc S}.\sigma$    & multisignature \\ \hline
+			\end{tabular}
+		\end{center}
+		The function $\Sno(v,n,T,U,\red{U_\alpha}, U_\omega)$ initializes a snapshot object with no multi-signature set.
```

Comment:
Must: fix inconsistency in argument ordering and what to pass here and in all later mentions of this.

Here it is given Utxos, in the off-chain figure (on ack sn) it is given transactions, and in the description of handling Ack Sn it is given utxo hashes $\eta$.